src/util-lua-tls.c

```diff
@@ -55,7 +55,33 @@
 #include "util-lua-common.h"
 #include "util-lua-tls.h"
 
-static int GetCertNotBefore(lua_State *luastate, const Flow *f, int direction)
+static const char tls_flow[] = "suricata:tls";
```

Comment:
strange name of var

Reply:
I'll update to match other lua libraries.